plugins/dev9ghzdrk/PS2Edefs.h

```diff
@@ -48,12 +48,25 @@
 
 #include "PS2Etypes.h"
 
-#ifdef __linux__
-#define CALLBACK
-#else
+#ifndef __linux__
 #include <windows.h>
 #endif
 
+#if defined(_MSC_VER)
+    //  Microsoft
+    #define EXPORT __declspec(dllexport)
+    #define IMPORT __declspec(dllimport)
```

Comment:
I don't think `IMPORT` is used anywhere?

Reply:
You are correct, it's defined but not necessary